docs/rst/user_manual/layout.rst

```diff
@@ -0,0 +1,175 @@
+
+.. todo::
+    Include a screenshot.
+
+**Logical** entities refer to the DDS *Domain* and *Topic* entities.
+For the explanation of this entities and the interaction with them refer to the section :ref:`logical_panel`.
+
+**Physical** entities refer to the main information of the machine and the context where a *DomainParticipant*
+is being executed.
+This information is divided in *Host*, *User* and *Process*.
+For the explanation of this entities and the interaction with them refer to the section :ref:`physical_panel`.
+
+.. _right_sidebar_layout:
+
+Right Sidebar
+=============
+
+.. _info_panel_layout:
+
+Info Panel
+----------
+
+This panel shows the main information of the last entity clicked.
+This information differs depending on the kind of the entity.
+i.e. For a *DDS Entity*, the *QoS* information would be shown, while for a *Process* it will be displayed the *pid*.
+
+.. todo::
+    Include a screenshot.
+
+For the explanation of this information refer to the section :ref:`info_panel`.
+
+.. _statistics_panel_layout:
+
+Statistics Panel
+----------------
+
+This panel shows a summary of the main statistic data related with the entity clicked.
+
+.. todo::
+    Include a screenshot.
+
+For the explanation of this information refer to the section :ref:`statistics_panel`.
+
+Issues Panel
+------------
+
+This panel shows a collection of news and issues that the monitor has received.
+The information stored is divided in three groups:
+
+- Callbacks that are received related with new entities discovered in the network.
+  This callbacks are erased every time *Refresh* is pressed.
+
+- Domains that are being monitoring at the moment.
+- Issues and problems automatically detected in the actual execution.
+
+.. todo::
+    Include a screenshot.
+
+For the explanation of this information refer to the section :ref:`statistics_panel`.
+
+.. _main_panel_layout:
+
+Main Panel
+==========
+
+In the central window it will be displayed the data charts or *Chartbox* that the user has configured.
+To create a new chart use the button *ApplicationMenu->Edit->DisplayNewData*. This button will create a new
+*Chartbox* where new series could be added, removed or modified.
+
+.. todo::
+    Include a screenshot.
+
+For further information about how to create a chart refer to the section :ref:`chart_panel`.
+
+.. _chartbox_layout:
+
+Chartbox
+--------
+These windows in the central panel store some *series* or *data configurations* to show a specific data type for
+one or several entities in a time interval and with different accumulative processes.
+
+These Chartbox could be moved along the main window.
+In order to move one of this charts, press in the Chart title and grab the object to the new location inside the
+main panel.
+The rest of the panels will automatically rearrange when one panel is being moved.
+
+.. todo::
+    Include a screenshot.
+
+.. _create_new_series_layout:
+
+Create new Serie Dialog
```

Comment:
My idea was to add here all the application screenshots, and so it would be better to have this section.
In case we add screenshots along the rest of the document, it is okey to remove it.